lib/token.cpp

```diff
@@ -29,6 +29,7 @@
 #include <algorithm>
 #include <cassert>
 #include <cctype>
+#include <cstdio>
```

Comment:
?

Reply:
It seems my editor sometimes insert these headers. I'll remove them.